webapp/channels/src/store/index.ts

```diff
@@ -84,7 +105,7 @@ export default function configureStore(preloadedState, additionalReducers) {
                     payload,
                 });
             },
-        });
+        } as any);
```

Comment:
Do we have a better type here? `as any` kind of removes any type checking.

Reply:
automatically inferring it was enough, and the code gets simpler.